pysat/tests/test_utils_io.py

```diff
@@ -166,7 +166,7 @@ def test_inst_write_and_read_netcdf(self):
 
         # Load the written file directly into an Instrument
         netcdf_inst = pysat.Instrument(
-            'pysat', 'netcdf', directory_format=file_path,
+            'pysat', 'netcdf', data_dir=file_path,
```

Comment:
Is this correcting a bug or do we need a deprecation warning?

Reply:
Correcting a bug. Also changed in #983 but it popped up in my local testing. While `directory_format` still exists and can be used to switch directories, that is technically hacking the keyword. `data_dir` was added at some point for that desired feature.